cmd/sync.go

```diff
@@ -115,6 +139,19 @@ func (raw *rawSyncCmdArgs) parsePatterns(pattern string) (cookedPatterns []strin
 	return
 }
 
+func (raw *rawSyncCmdArgs) parseCFDMode() common.CFDMode {
+	cfdMode := strings.ToLower(raw.cfdMode)
+	if cfdMode == strings.ToLower(common.CFDModeFlags.TargetCompare().String()) {
+		return common.CFDModeFlags.TargetCompare()
+	} else if cfdMode == strings.ToLower(common.CFDModeFlags.Ctime().String()) {
+		return common.CFDModeFlags.Ctime()
+	} else if cfdMode == strings.ToLower(common.CFDModeFlags.CtimeMtime().String()) && raw.lastSyncTime != "" {
```

Comment:
instead of doing the lastSyncTime check here, let the caller do it. It can log an appropriate message/warning too.
Also, lastSyncTime should be valid for both CtimeMtime and Ctime cfd modes, not just CtimeMtime

Reply:
sorry my bad, i don't want to check at all if time is nil or not. Reason for that customer want to try sync as first copy only then in that case lastSyncTime should be zero. I will add the comment in argument side for cfdMode.